### PR TITLE
Fixing up small bugs; run for 50 model instances.

### PR DESCRIPTION
**Pull request by bcipolli**
@vishaalprasad please take a look and let me know if you have any questions!

Fixed this up so we could run a sufficient # of models to get some results. Results should be visible on guru at `~bcipolli/code/de/results/` in the morning!

### CODE REVIEW

File: code/_expts/slotnick_etal_2001/analyze/plots/de_CreateSlotnickFigure1.m [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
+% de_CreateSlotnickFigure1(Y1, E1)
 %  Y1:  errorbar y
 %  E1:  errorbar e
-
-%  Auto-generated by MATLAB on 31-Oct-2015 17:57:12
+%  taskType: slotnick task
 
 % Create figure
-figure1 = figure;
+figure1 = de_NewFig(sprintf('blobdot_%s_', taskType));
```
Comment (bcipolli) [inline code/_expts/slotnick_etal_2001/analyze/plots/de_CreateSlotnickFigure1.m]: arg is the plot name, which is a tag used for saving the figure to disk.

**Comment by vishaalprasad** [comment]
Looks good to me!

**Comment by bcipolli** [comment]
Hmmm, sigmas look like they are `5` and `25`...
image

**Comment by bcipolli** [comment]
Code is not running headless on `guru`; looks like the plotting code still exists in `de_StimCreate`, @vishaalprasad ? I will take a look.
image

**Comment by bcipolli** [comment]
@vishaalprasad this is running now on `guru` after I removed the rest of the plotting code from `de_StimCreate`; will merge.

**Pull request merged**